custom_components/petlibro/api.py

```diff
@@ -335,6 +335,9 @@ async def device_grain_status(self, serial: str) -> Dict[str, Any]:
 
     async def device_feeding_plan_today_new(self, serial: str) -> Dict[str, Any]:
         return await self.session.post_serial("/device/feedingPlan/todayNew", serial)
+    
```

Comment:
This is similar to what is outlined in pull request [#47](https://github.com/jjjonesjr33/petlibro/pull/47), but presented in a different format.

Code from pull request [#47](https://github.com/jjjonesjr33/petlibro/pull/47)
```python
    async def device_wet_feeding_plan(self, serial: str) -> Dict[str, Any]:
        return await self.session.post_serial("/device/wetFeedingPlan/wetListV3", serial)
```

Refer to comment in your `polar_wet_food_feeder.py`